[2023-08-18T04:56:50.938+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dags_external_task_sensor.external_task_sensor_a manual__2023-08-18T04:55:47.721833+00:00 [queued]>
[2023-08-18T04:56:50.953+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dags_external_task_sensor.external_task_sensor_a manual__2023-08-18T04:55:47.721833+00:00 [queued]>
[2023-08-18T04:56:50.954+0000] {taskinstance.py:1308} INFO - Starting attempt 2 of 2
[2023-08-18T04:56:50.974+0000] {taskinstance.py:1327} INFO - Executing <Task(ExternalTaskSensor): external_task_sensor_a> on 2023-08-18 04:55:47.721833+00:00
[2023-08-18T04:56:50.983+0000] {standard_task_runner.py:57} INFO - Started process 2619 to run task
[2023-08-18T04:56:50.986+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dags_external_task_sensor', 'external_task_sensor_a', 'manual__2023-08-18T04:55:47.721833+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/dags_external_task_sensor.py', '--cfg-path', '/tmp/tmp43gmdozh']
[2023-08-18T04:56:50.987+0000] {standard_task_runner.py:85} INFO - Job 40: Subtask external_task_sensor_a
[2023-08-18T04:56:51.044+0000] {task_command.py:410} INFO - Running <TaskInstance: dags_external_task_sensor.external_task_sensor_a manual__2023-08-18T04:55:47.721833+00:00 [running]> on host 05c492c53b17
[2023-08-18T04:56:51.147+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dags_external_task_sensor' AIRFLOW_CTX_TASK_ID='external_task_sensor_a' AIRFLOW_CTX_EXECUTION_DATE='2023-08-18T04:55:47.721833+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-18T04:55:47.721833+00:00'
[2023-08-18T04:56:51.148+0000] {external_task.py:236} INFO - Poking for tasks ['task_a'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:57:01.173+0000] {external_task.py:236} INFO - Poking for tasks ['task_a'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:57:11.194+0000] {external_task.py:236} INFO - Poking for tasks ['task_a'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:57:21.217+0000] {external_task.py:236} INFO - Poking for tasks ['task_a'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:57:31.236+0000] {external_task.py:236} INFO - Poking for tasks ['task_a'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:57:41.256+0000] {external_task.py:236} INFO - Poking for tasks ['task_a'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:57:51.276+0000] {external_task.py:236} INFO - Poking for tasks ['task_a'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:58:01.296+0000] {external_task.py:236} INFO - Poking for tasks ['task_a'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:58:11.316+0000] {external_task.py:236} INFO - Poking for tasks ['task_a'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:58:21.336+0000] {external_task.py:236} INFO - Poking for tasks ['task_a'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:58:31.356+0000] {external_task.py:236} INFO - Poking for tasks ['task_a'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:58:41.377+0000] {external_task.py:236} INFO - Poking for tasks ['task_a'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:58:51.398+0000] {external_task.py:236} INFO - Poking for tasks ['task_a'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:58:57.127+0000] {local_task_job_runner.py:292} WARNING - State of this instance has been externally set to queued. Terminating instance.
[2023-08-18T04:58:57.132+0000] {process_utils.py:135} INFO - Sending Signals.SIGTERM to group 2619. PIDs of all processes in the group: [2619]
[2023-08-18T04:58:57.133+0000] {process_utils.py:86} INFO - Sending the signal Signals.SIGTERM to group 2619
[2023-08-18T04:58:57.134+0000] {taskinstance.py:1517} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-08-18T04:58:57.156+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/sensors/base.py", line 253, in execute
    time.sleep(self._get_next_poke_interval(started_at, run_duration, try_number))
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1519, in signal_handler
    raise AirflowException("Task received SIGTERM signal")
airflow.exceptions.AirflowException: Task received SIGTERM signal
[2023-08-18T04:58:57.167+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=dags_external_task_sensor, task_id=external_task_sensor_a, execution_date=20230818T045547, start_date=20230818T045650, end_date=20230818T045857
[2023-08-18T04:58:57.201+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 40 for task external_task_sensor_a (Task received SIGTERM signal; 2619)
[2023-08-18T04:58:57.252+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=2619, status='terminated', exitcode=1, started='04:56:50') (2619) terminated with exit code 1
